internal/uvm/scsi.go

```diff
@@ -331,13 +350,144 @@ func (uvm *UtilityVM) GetScsiUvmPath(ctx context.Context, hostPath string) (stri
 }
 
 // grantAccess helper function to grant access to a file for the vm or vm group
-func (uvm *UtilityVM) grantAccess(ctx context.Context, hostPath string, vmAccess VMAccessType) error {
+func grantAccess(ctx context.Context, uvmID string, hostPath string, vmAccess VMAccessType) error {
 	switch vmAccess {
 	case VMAccessTypeGroup:
 		log.G(ctx).WithField("path", hostPath).Debug("granting vm group access")
 		return security.GrantVmGroupAccess(hostPath)
 	case VMAccessTypeIndividual:
-		return wclayer.GrantVmAccess(ctx, uvm.id, hostPath)
+		return wclayer.GrantVmAccess(ctx, uvmID, hostPath)
 	}
 	return nil
 }
+
+var _ = (Cloneable)(&SCSIMount{})
+
+// serializes the SCSIMount struct
```

Comment:
Start these comments with the name of the exported method to make go smile down on us. Excerpt of what this is going to look like in vscode with go-lint installed :P

`comment on exported method SCSIMount.GobEncode should be of the form "GobEncode ..."go-lint`